### PR TITLE
Fix etcd listening address on master nodes

### PR DESCRIPTION
**Pull request by jchauncey**
This removes the public ip interface from the set of listening addresses for etcd. 

cc @jdumars @anhowe @jackfrancis

### CODE REVIEW

File: examples/kubernetes.json [hunk 1]
```diff
@@ -5,7 +5,7 @@
       "orchestratorType": "Kubernetes"
     },
     "masterProfile": {
-      "count": 1,
+      "count": 3,
```
Comment (jackfrancis) [inline examples/kubernetes.json]: why this change?

Comment (jchauncey) [reply examples/kubernetes.json]: sorry got caught up in the commit didnt mean to push it. just fixed it
